wgsl/index.bs

```diff
@@ -2082,6 +2082,18 @@ shift_expression
   | shift_expression SHIFT_RIGHT additive_expression
         OpShiftRightLogical or OpShiftRightArithmetic
 
+and_expression
+  : equality_expression
+  | and_expression AND equality_expression
+
+exclusive_or_expression
+  : and_expression
+  | exclusive_or_expression XOR and_expression
+
+inclusive_or_expression
+  : exclusive_or_expression
+  | inclusive_or_expression OR exclusive_or_expression
+
 relational_expression
   : shift_expression
```

Comment:
This should point to `inclusive_or_expression`

Reply:
Done!